great_ape_safe/ape_api/pancakeswap_v2.py

```diff
@@ -0,0 +1,15 @@
+from brownie import web3
+from great_ape_safe.ape_api.uni_v2 import UniV2
+from helpers.addresses import registry
+
+
+class Pancakeswap(UniV2):
```

Comment:
would stick to convention of `PancakeswapV2` or `PancakeV2`.

i say this also because i know we will need to add v1 at one point, since some of our lps on bsc will need v1 and some will need v2 router to withdraw from: https://bscscan.com/tokenholdings?a=0x6DA4c138Dd178F6179091C260de643529A2dAcfe

Reply:
really smooth use of inheritance here btw 🤩

Reply:
last commit adapts better naming for the versions all around